project/src/components/my-list-screen/my-list-screen.tsx

```diff
@@ -1,133 +1,36 @@
-function MyListscreen(): JSX.Element {
+import Logo from '../logo/logo';
+import type { Film } from '../../types/types';
+import UserBlock from '../user-block/user-block';
+import FilmCardsList from '../catalog-films-list/catalog-films-list';
+import PageFooter from '../page-footer/page-footer';
+import Catalog from '../catalog/catalog';
+import PageTitle from '../title/title';
+import PageHeader from '../header/header';
+
+type MyListScreenProps = {
+  films: Film[],
+}
+
+function MyListScreen({films}: MyListScreenProps): JSX.Element {
+  const getFavoriteFilms = () => films.filter((film) => film.isFavorite);
```

Comment:
```suggestion
  const getFavoriteFilms = (films) => films.filter((film) => film.isFavorite);
```
и это тоже лучше вынести, чтобы компонент был чище